apps/src/aichat/views/ChatMessage.tsx

```diff
@@ -24,7 +24,11 @@ const isUser = (role: string) => role === Role.USER;
 const isModelUpdate = (role: string) => role === Role.MODEL_UPDATE;
 
 const displayUserMessage = (status: string, chatMessageText: string) => {
-  if (status === Status.OK || status === Status.UNKNOWN) {
+  if (
+    status === Status.OK ||
+    status === Status.UNKNOWN ||
+    status === Status.ERROR
```

Comment:
If we mark the response from the model in an error state, we also mark the preceeding student message as in an "error" state, such that it doesn't get sent to the model on the next request cycle (we only send messages in the "ok" state, in addition to the current message from the user). From a UI perspective, though, treating it like other messages. Curious for other thoughts on a) whether that UI seems ok, and b) whether marking it as an "error" message seems ok.

Reply:
Raised here, no objections: https://codedotorg.slack.com/archives/C06FELVTV0Q/p1714773355042179

Reply:
Discussed briefly last week - I do think it feels a little weird to call this an "Error" state when it wasn't truly a user error, but also understand the need for not marking it "OK" so it gets filtered out. Not sure if I have a better idea of what to use and I don't want to add another constant just for the sake of it, so fine with this for now :) happy to revisit model architecture more broadly on Slack/our sync